aidatlu/constellation/README.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "AidaTLU"
+description: "Satellite for the AIDA-2020 TLU using a Python based control software"
+---
+
+## Description
+
+The AIDA-2020 Trigger Logic Unit is designed to provide flexible trigger configurations in test beam setups (https://doi.org/10.48550/arXiv.2005.00310).
+
+The Python-based control software for the AIDA-2020 TLU provides a comprehensive interface for controlling the TLU.
+The software establishes a connection to the hardware and allows for easy configuration of different trigger setups.
+Information over each individual trigger signal is saved in a compressed and human-readable HDF5 format.
+
+The satellite connects the AIDA-2020 TLU to the [Constellation](https://constellation.pages.desy.de/) control and data acquisition framework.
+
+
+## Building
+
+After installing [IPbus](https://ipbus.web.cern.ch/doc/user/html/software/install/compile.html), with Python bindings (uhal), install the [Aida-TLU](https://github.com/SiLab-Bonn/aidatlu) package with the constellation requirement.
+
+```bash
+pip install .[constellation]
+```
+
+A more detailed description of the prerequisites can also be found [here](https://silab-bonn.github.io/aidatlu/Introduction.html#installation).
+
+## Usage
+
+Add the chosen cactus library path, where the default installation location is `/opt/cactus/`:
+
+```bash
+export LD_LIBRARY_PATH=<install_location>/lib
+```
+
+You also need to start the control hub:
+```bash
+<install_location>/bin/controlhub_start
+```
+
+Finally, start the satellite with:
+```bash
+SatelliteAidaTLU -g testbeam -n TLU
+```
+
+The AIDA-TLU satellite can be tested using a
+TLU mock. By default, the hardware will be used, but the mock can be selected using an environmental variable.
+
+```bash
+TLU_MOCK=True SatelliteAidaTLU -g testbeam -n TLU
+```
+
+## Parameters
+
```

Comment:
Could you indicate which parameters are required and which are optional? Some might benefit from a default, e.g. the clock lemo out which is rarely used, could default to `False`.